routes/index.js

```diff
@@ -1,9 +1,58 @@
 var express = require('express');
 var router = express.Router();
+var path = require('path');
+var User = require(path.join(__dirname,'../models/studentDataModel')).user;
+var Attendence = require(path.join(__dirname,'../models/studentDataModel')).attendence;
+var Data = require(path.join(__dirname,'../models/studentDataModel')).data;
+var Student = require(path.join(__dirname,'../models/studentDataModel')).student;
+var Grades = require(path.join(__dirname,'../models/studentDataModel')).grades;
+
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
   res.render('index', { title: 'Express' });
 });
 
+router.post('/login', function(req, res){
+	//POST login
+	res.sendfile('./views/index.html');
```

Comment:
According to [this](http://stackoverflow.com/questions/25463423/res-sendfile-absolute-path) stackoverflow: `res.sendfile` is deprecated. I think the currently supported way to do it is:

```
res.sendFile('index.html', { root: path.join(__dirname, '../views') });
```

I think this should work?